.github/workflows/build_notarized.yml

```diff
@@ -349,9 +349,11 @@ jobs:
         echo "🔗 [${DMG_S3_PATH}](${DMG_URL})" >> $GITHUB_STEP_SUMMARY
         echo "🔗 [${DSYM_S3_PATH}](${DSYM_URL})" >> $GITHUB_STEP_SUMMARY
 
-        bundle exec fastlane run asana_add_comment \
-          task_url:"${{ env.asana-task-url }}" \
-          comment:"New build is available at ${DMG_URL}."
+        if [[ -n "${{ env.asana-task-url }}" ]]; then
```

Comment:
Skip posting a comment to Asana when Asana task is not provided to the workflow. The code just above generates the workflow summary which we're interested in also when Asana task URL is not present.